Multisig/Logic/Models/KeyInfo.swift

```diff
@@ -37,6 +55,11 @@ extension KeyInfo {
         set { type = Int16(newValue.rawValue) }
     }
 
+    var web3AuthType: LoginType {
+        get { LoginType(rawValue: Int(loginType)) ?? .none }
```

Comment:
These two line don't compile for me. 

`/Users/biafra/safe-ios/Multisig/Logic/Models/KeyInfo.swift:59:39 Cannot find 'loginType' in scope`